lucille-plugins/lucille-weaviate/src/main/java/com/kmwllc/lucille/weaviate/connector/WeaviateConnector.java

```diff
@@ -0,0 +1,18 @@
+package com.kmwllc.lucille.weaviate.connector;
+
+import com.kmwllc.lucille.connector.AbstractConnector;
+import com.kmwllc.lucille.core.ConnectorException;
+import com.kmwllc.lucille.core.Publisher;
+import com.typesafe.config.Config;
+
+public class WeaviateConnector extends AbstractConnector {
```

Comment:
Let's remove this from the PR. It's not implemented and is not being used anywhere (but let me know if I'm missing something).

Reply:
I was thinking of moving the code that retrieves ids in the DropDocument stage in this PR into the a separate weaviate connector and to retire the stage